frontend/src/Store/Actions/Settings/importLists.js

```diff
@@ -107,8 +107,8 @@ export default {
 
     [SELECT_IMPORT_LIST_SCHEMA]: (state, { payload }) => {
       return selectProviderSchema(state, section, payload, (selectedSchema) => {
-        selectedSchema.enableRss = selectedSchema.supportsRss;
-        selectedSchema.enableSearch = selectedSchema.supportsSearch;
+        selectedSchema.enableAutomaticAdd = true;
```

Comment:
Nice catch on this old copy paste, but don't think there is any need to explicitly set things here, these should already default to false and none. Unless we think we want these as default?